recipes/umap-learn/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "umap-learn" %}
+{% set version = "0.2.3" %}
+{% set sha256 = "87012d966a61d79cad81703dd1a46c4a48d9a602918899f228b67ac6bc98abc5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win and py27 or win32]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - numpy >=1.13
+    - numba >=0.35
+    - scikit-learn >=0.19
+    - scipy >=0.18
+
+test:
+  imports:
+    - umap
+  requires:
+    - nose
+
+about:
+  home: http://github.com/lmcinnes/umap
+  license: BSD-2-Clause
+  license_family: BSD
+  # license_file: LICENSE.txt
```

Comment:
Please manually package the license and consider adding it to the source distribution:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually